source/Ship.cpp

```diff
@@ -4224,6 +4231,65 @@ const vector<weak_ptr<Ship>> &Ship::GetEscorts() const
 
 
 
+vector<pair<string, CaptureClass>> Ship::CaptureRequirements() const
```

Comment:
Usually I dont approve of putting comments in both the cpp and h but in this case since its done for other functions and can be a bit of a complex return type (especially for the second function) I'd add it here too?